### ISSUE TITLE
Add tool to get lumi json for lumi calc

### ISSUE THREAD

**Issue by lcorcodilos** [lcorcodilos/TIMBER#9]
Port [printLumiJson.py](https://github.com/lcorcodilos/Bstar13TeV-AllHad/blob/master/lumi_jsons/printLumiJson.py)

**Comment by lcorcodilos** [comment]
Also add the following code (but note the need to add boost as a dependency),

```c++
#include <boost/property_tree/ptree.hpp>
#include <boost/property_tree/json_parser.hpp>
#include <vector>
#include <string>
#include <algorithm>
#include <iostream>

class LumiFilter
{
private:
    boost::property_tree::ptree ptree;
public:
    LumiFilter(std::string filename = "/cms/xaastorage/PicoTrees/JSON_FILES/Cert_271036-284044_13TeV_ReReco_07Aug2017_Collisions16_JSON.txt");
    ~LumiFilter();
    float Eval(float run, float lumi);
};

LumiFilter::LumiFilter(std::string filename = "/cms/xaastorage/PicoTrees/JSON_FILES/Cert_271036-284044_13TeV_ReReco_07Aug2017_Collisions16_JSON.txt") {
    boost::property_tree::read_json(filename, ptree);
}

LumiFilter::~LumiFilter() {
}

float LumiFilter::Eval(float run, float lumi) {
    for ( const auto& runEntry : ptree ) {
        if (std::stoul(runEntry.first) == (unsigned int)run) {
            for ( const auto& lrEntry : runEntry.second ) {
                const auto lrNd = lrEntry.second;
                unsigned int lumStart = std::stoul(lrNd.begin()->second.data());
                unsigned int lumEnd = std::stoul((++lrNd.begin())->second.data());
                unsigned int L = (unsigned int) lumi;
                if ((L < lumEnd) && (L > lumStart)) {
                    return 1.;
                }
            }
        }
    }
    return 0.;
}
```

**Comment by lcorcodilos** [comment]
Adapted version of copied script (`LumiFilter`) included in #19. Still need to port printLumiJson.py
